web/source/osk/embedded/subkeyDelegator.ts

```diff
@@ -52,5 +60,25 @@ namespace com.keyman.osk.embedded {
       }
       this.resolver = null;
     }
+
+    public isVisible(): boolean {
+      return true;
+    }
+
+    public clear() {
+      // no-op; it's fully controlled on the app side.
+    }
+
+    updateTouch(touch: Touch) {
+      let baseKeyTouched = this.baseKey.key.isUnderTouch(touch);
+      this.baseKeySelected = this.baseKey.key.isUnderTouch(touch)
+
+      // Prevent highlighting & selection before the touch has moved from the base key.
+      if(this.movedFromBaseKey) {
+        this.baseKey.key.highlight(this.baseKeySelected);
+      } else {
+        this.movedFromBaseKey = !baseKeyTouched;
+      }
+    }
```

Comment:
This part... is technically a bug fix.  Current versions of Keyman for Android actually can't return the base key if you return your finger to it after the subkey menu has been displayed... let alone highlight that base key.